ocaml/xapi/xapi_vm_lifecycle.ml

```diff
@@ -629,6 +629,11 @@ let force_state_reset_keep_current_operations ~__context ~self ~value:state =
            Db.PCI.set_scheduled_to_be_attached_to ~__context ~self:pci ~value:Ref.null
       )
       (Db.PCI.get_all ~__context);
+    List.iter
+      (fun vusb ->
+         Db.VUSB.set_currently_attached ~__context ~self:vusb~value:false;
```

Comment:
add a space

Reply:
Have you not even compiled this?

Reply:
I compiled this. It works. I just go through again the code and find this.

Reply:
every push I will compile the code.

Reply:
done